### PR TITLE
Added debounce to set temperature

### PR DESCRIPTION
**Pull request by igbopie**

### CODE REVIEW

File: lib/nest-thermostat-accessory.js [hunk 1]
```diff
@@ -213,7 +223,7 @@ NestThermostatAccessory.prototype.setTargetTemperature = function (targetTempera
 	return this.cancelAutoAway()
 		.then(this.updateDevicePropertyAsync.bind(this, key, targetTemperature, prop + "target temperature"))
 		.asCallback(callback);
-};
+}, 5000);
```
Comment (clintonb) [inline lib/nest-thermostat-accessory.js]: A 5s delay is right on the cusp of making me question if I actually completed an action. Consider making this user-configurable. Alternatively, decrease the delay, perhaps to 2 or 3s.

**Comment by clintonb** [comment]
This resolves https://github.com/KraigM/homebridge-nest/issues/83.

**Comment by clintonb** [comment]
@KraigM can we get this merged, and released, pleased?

**Comment by idcrook** [comment]
thanks for making this available. I was able to use by pointing npm to the repo `sudo npm install -g git+https://github.com/igbopie/homebridge-nest.git` on my raspberry pi

**Comment by ozaoral** [comment]
tested, works well on a setup that previously disconnected (after trying to change the temperature) and reconnected few minutes later again

**Comment by congsun** [comment]
FYI, I've pushed a new npm package ```homebridge-nest-cong``` for everyone's convenience. It has the debounce with the delay of 3s. Thanks for making this available!

**Pull request merged**